docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,11 @@ This page contains release notes for the SDK.
 HEAD — ongoing
```

Comment:
Release notes should say something about 1.4 since its something users end up seeing.

Reply:
Added a not.